params.yaml

```diff
@@ -64,16 +63,15 @@ input:
   complex:
     # Townhomes (class 210/295) should match exactly on these variables to be
     # considered in the same complex
-    match_exact: [
-      "meta_township_code",
-      "meta_class",
-      "char_bsmt",
-      "char_gar1_size",
-      "char_attic_fnsh",
-      "char_beds",
-      "meta_pin_num_cards",
-      "meta_tieback_proration_rate"
-    ]
+    match_exact:
+      - "meta_township_code"
+      - "meta_class"
+      - "char_bsmt"
+      - "char_gar1_size"
+      - "char_attic_fnsh"
+      - "char_beds"
+      - "meta_pin_num_cards"
+      - "meta_tieback_proration_rate"
```

Comment:
Just cleaning up the parameter formatting using actual YAML arrays. No content changes here.